crates/ruff/src/registry/rule_set.rs

```diff
@@ -7,10 +7,10 @@ use std::iter::FusedIterator;
 ///
 /// Uses a bitset where a bit of one signals that the Rule with that [u16] is in this set.
 #[derive(Clone, Default, CacheKey, PartialEq, Eq)]
-pub struct RuleSet([u64; 9]);
+pub struct RuleSet([u64; 10]);
```

Comment:
Is this okay? It fixed a test failure.

Reply:
Yeah, this is fine. I assume your rule increased ruff's rule count over 576